src/utils/isNaN.ts

```
@@ -0,0 +1,3 @@
+export const isNaN = (value: unknown): boolean => {
```

Biome (1.9.4)

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)